src/test/java/org/codeforamerica/shiba/output/MnitDocumentConsumerTest.java

```diff
@@ -75,7 +75,7 @@ public ApplicationData applicationData() {
     private DocumentRepositoryService documentRepositoryService;
     @MockBean
     private FileNameGenerator fileNameGenerator;
-    @MockBean
+    @SpyBean
     private ApplicationRepository applicationRepository;
```

Comment:
Huh, `@MockBean` doesn't work with h2?

Reply:
I thought we couldn't at the time of this change but it seems to be actually fine now. Good callout haha. I'll revert it to `@MockBean`